shared/src/main/scala/squants/market/Money.scala

```diff
@@ -14,6 +14,7 @@ import scala.util.{Failure, Success, Try}
 import scala.language.implicitConversions
 import scala.math.BigDecimal.RoundingMode
 import scala.math.BigDecimal.RoundingMode.RoundingMode
+import java.util.Objects
```

Comment:
This is import is unused and it is breaking the build

Reply:
I removed the `java.util` package from the hash function.